### PR TITLE
Create method continued

### PR DESCRIPTION
**Pull request by Aferencz1987**
Iteration 0 functionality of merchant and merchant repo completed.

### CODE REVIEW

**Comment by matthewjholmes** [comment]
Thanks for your work. Looks good.

File: lib/merchant_repository.rb [hunk 1]
```diff
@@ -38,19 +38,22 @@ def find_all_by_name(name)
...
   end
 
-  def self.create(attributes)
-    # id = attributes[:id]
+  def create(attributes)
     name = attributes[:name]
-    # id = new_id_number
-    # require 'pry'; binding.pry
-    Merchant.new({:id => new_id_number, :name => name})
-
```
Comment (lcpulzone) [inline lib/merchant_repository.rb:47]: Woohoo!  Glad this is working.

**Review by lcpulzone — COMMENTED** [review]
Looks great, refactor here we come.

**Review by aliyamerali — APPROVED** [review]
Looks great!

**Pull request merged**